examples/ota-requestor-app/linux/README.md

```diff
@@ -22,6 +22,7 @@ following command line options are available for the OTA Requestor application.
 | -p/--periodicQueryTimeout <Time in seconds> | Periodic timeout for querying providers in the default OTA provider list. If none or zero is supplied the timeout is set to every 24 hours. |
 | -c/--requestorCanConsent                    | If supplied, the RequestorCanConsent field of the QueryImage command is set to true. Otherwise, the value is determined by the driver.      |
 | -f/--otaDownloadPath <file path>            | If supplied, the OTA image is downloaded to the given fully-qualified file-path. Otherwise, the value defaults to /tmp/test.bin.            |
+| -u/--userConsentState <granted              | denied                                                                                                                                      | deferred> | The following is applied to the next QueryImage and will revert back to the behavior of granted on subsequent QueryImage commands. <li> granted: Authorize OTA requestor to download an OTA image. <li> denied: Forbid OTA requestor to download an OTA image. <li> deferred: Defer obtaining user consent. |
```

Comment:
I think saying revert back is confusing as the concept of revert doesn't make sense. It's not like they got to specify one behavior before this and it reverts back. Maybe better to just mention default values:
```suggestion
| -u/--userConsentState <granted              | denied                                                                                                                                      | deferred> | The user consent state for the first QueryImage command. For all subsequent commands, the value of granted will be used. <li> granted: Authorize OTA requestor to download an OTA image. <li> denied: Forbid OTA requestor to download an OTA image. <li> deferred: Defer obtaining user consent. |
```